lua/autorun/server/sv_cfc_contact_forms.lua

```diff
@@ -145,31 +145,42 @@ local function sendFailureAlert( ply )
     net.Send( ply )
 end
 
-local function submitFormForPlayer( data, endpoint, ply )
-    local plyName = ply and ply:GetName() or "Unknown Player"
+util.AddNetworkString( "cfc_contact_forms_alert" )
```

Comment:
need to make sure you change this too to use the new net message name

Reply:
Fixed. 👍